examples/lighting-app/nrfconnect/CMakeLists.txt

```diff
@@ -114,6 +114,10 @@ set(PIGWEED_ROOT "${CHIP_ROOT}/third_party/pigweed/repo")
 include(${PIGWEED_ROOT}/pw_build/pigweed.cmake)
 include(${PIGWEED_ROOT}/pw_protobuf_compiler/proto.cmake)
 
+# TODO: Remove this temporary fix needed to get RPC builds working.
```

Comment:
Hmm. But we're verifying this build in CI and it currently passes. In which conditions in doesn't? Also, note that we're adding `target_link_libraries(pw_build INTERFACE zephyr_interface)` below which should add these flags. Does Pigweed contain any targets which don't depend on `pw_build`?

Reply:
It doesn't actually fail to compile, but the RPCs are not working. I'm still trying to figure this out with the Pigweed team, just wanted to put up a fix while I continue to debug.

Reply:
I  see. This reminds me one past issue where also some incompatible flags were set for building: https://github.com/project-chip/connectedhomeip/pull/7779. If now something like that happens it's probably that we're building something not dependent on `pw_build`.

Reply:
Thanks for the link!